UnitsNet/UnitConverter.cs

```diff
@@ -397,9 +407,10 @@ private static bool TryParseUnit(Type unitType, string unitName, out object unit
 
         private static bool TryGetUnitType(string quantityName, out Type unitType)
         {
-            string unitTypeName = $"{UnitTypeNamespace}.{quantityName}Unit";
+            quantityName += "Unit";
```

Comment:
Make it a variable, f.ex. `unitTypeName`. Also I think we can keep the example comment, but modify to `// Example: "LengthUnit"`. I think this is helpful when reading the code.